### DISCUSSION TITLE
Show queue whenever a new song plays

### DISCUSSION THREAD

**Discussion by compmix** [jagrosh/MusicBot#1366] [Ideas]
Hi there, awesome, simple bot!

It would be nice to have a toggle that whenever the next song plays, it also shows the nowplaying bar and queue automatically.

Thanks!

**Comment by bebida11** [comment]
Listo! Ahora ha descargado y configurado un nuevo tono de llamada desde descargartonosdellamada.com. Disfruta de tu tono de llamada personalizado y añade un toque único a tus llamadas entrantes. Recuerda que puedes repetir estos pasos para [descargar tonos de llamada](https://descargartonosdellamada.com/) y cambiarlos segun tus preferencias. ¡Diviértete personalizando tu experiencia sonora
